factory/factory.go

```diff
@@ -12,48 +12,41 @@ package factory
 
 import (
 	"fmt"
+	"net/url"
 	"os"
 
-	protos "github.com/omec-project/config5g/proto/sdcoreConfig"
 	"github.com/omec-project/udr/logger"
 	"gopkg.in/yaml.v2"
 )
 
 var UdrConfig Config
 
-type UpdateDb struct {
-	SmPolicyTable *SmPolicyUpdateEntry
-}
-
-type SmPolicyUpdateEntry struct {
-	Snssai *protos.NSSAI
-	Imsi   string
-	Dnn    string
-}
-
 // TODO: Support configuration update from REST api
 func InitConfigFactory(f string) error {
-	if content, err := os.ReadFile(f); err != nil {
+	content, err := os.ReadFile(f)
+	if err != nil {
 		return err
-	} else {
-		UdrConfig = Config{}
+	}
+	UdrConfig = Config{}
```

Comment:
Any reason for this variable to be uppercased?

```suggestion
	udrConfig = Config{}
```

Reply:
yes, it's a global variable used in other packages as `factory.UdrConfig`
(Legacy code)